api/v1alpha1/virtualmachinepublishrequest_types.go

```diff
@@ -215,6 +220,16 @@ type VirtualMachinePublishRequestStatus struct {
 	// +optional
 	StartTime metav1.Time `json:"startTime,omitempty"`
 
+	// Attempts represents the number of times the request has been attempted.
```

Comment:
Maybe qualify the comment a little more?  Same comment for L231.
```suggestion
	// Attempts represents the number of times the request to publish the VM has been attempted.
```